src/momentOfTruth.ts

```diff
@@ -180,7 +180,9 @@ export async function lintDiff(utils: TypeUtils, devOps: TypeDevOps) {
   const pr = await devOps.createPullRequestProperties(cli.defaultConfig());
   const configsToProcess = await utils.getConfigFilesChangedInPR(pr);
 
-  tagsMap = await utils.getTagsFromChangedFile(await utils.getChangedFilesFromPR(pr));
+  tagsMap = await utils.getTagsFromChangedFile(
```

Comment:
What's the difference between these functions `getChangedFilesFromPR` and `getFilesChangedInPR`?

Reply:
`getFilesChangedInPR` will filter example json, while `getChangedFilesFromPR ` not

Reply:
Can you rename them to reflect the differences?

Reply:
I 'll remove the `getChangedFilesFromPR ` function as it hasn't been referred any more